queries/runescript/highlights.scm

```diff
@@ -0,0 +1,100 @@
+; Variables
+[
+  (identifier)
+  (local_variable)
+  (game_variable)
+  (constant_variable)
+  (parameter_variable)
+  (declaration_variable)
+] @variable
+
+; Scripts
+(script
+  trigger: (identifier) @keyword.directive
+  subject: (identifier) @function)
+
+; Calls
+(call
+  [
+    "~"
+    "@"
+  ]? @function
+  callee: (identifier) @function)
+
+; Literals
+((literal) @boolean
+  (#any-of? @boolean "true" "false"))
+
+(literal
+  "null" @constant.builtin)
+
+(integer_literal) @number
+
+(coord_literal) @string.special
+
+(comment) @comment @spell
+
+(string_fragment) @string
```

Comment:
Also for `"\""`

Reply:
Actually I just meant we need a highlight for regular double quotes here (as string), which must be `"\"" @string` in the query file. Kind of confusing! But yes we should highlight escape sequences as well as `@string.escape`

Reply:
Ahh, makes sense! Yes, I will add both